### DISCUSSION TITLE
push error

### DISCUSSION THREAD

**Discussion by MihailNikolov04** [Cyfrin/foundry-full-course-cu#1197] [Q&A]
Hello, I have this  error when trying to push to github
 ```error: src refspec main does not match any error: failed to push some refs to 'https://github.com/MihailNikolov04/foundry-erc20-f23.git' ``` and I don't know what to do. Can someone please help?

**Comment by pratokko** [comment]
Hello, that means you have cloned the repo, and you have the .git file in the project what you have to do is find the project and delete the .git then you can push to GitHub

**Reply by MihailNikolov04** [reply]
Hi, and where exactly should this file be? There is a .git and .github folders and there is no such think as .git file

**Reply by MihailNikolov04** [reply]
thing*

**Reply by pratokko** [reply]
Delete the git folder and you will be able to push. After that reinitialize git , add, commit and push. That should work now.
